tuplex/codegen/include/BlockGeneratorVisitor.h

```diff
@@ -533,6 +541,22 @@ namespace codegen {
         // deal with the case when for loop expression is a tuple
         void visitUnrolledLoopSuite(NSuite*);
 
+        /*!
+         * extracted from visit(NFor *forStmt)
```

Comment:
extracted from?

Reply:
The function is a helper function for visiting NFor node. Updated in comments.